docs/glsl-fuzz-walkthrough.md

```diff
@@ -711,26 +715,18 @@ The results will be output to the current directory by default, or to the direct
 by `--output`.
 The output `.txt` files contain the run log for each shader.
 
-### vkrun
-
-As described earlier,
-the Vulkan worker uses `vkrun`
-to run SPIR-V shaders on the device.
-You can use this script directly
-if you want to re-run some SPIR-V shaders.
-The vertex shader must be provided explicitly, by default you can use our "passthrough"
-vertex shader available in
-`vulkan-worker/samples/shader.vert.spv`.
-Your device must show in `adb devices`.
-For example:
+### `runspv`
+
+As described earlier, the Vulkan worker uses `runspv` to run shaders on the
```

Comment:
I prefer to do "As described under [...](#internal-link)", to make our documentation more robust when we rearrange things.  I.e, avoid "earlier", "later", "above", "below".